src/schema/sale/index.js

```diff
@@ -90,15 +91,19 @@ export const SaleType = new GraphQLObjectType({
           request,
           { rootValue: { saleArtworksLoader } }
         ) => {
-          const invert = saleArtworks => map(saleArtworks, "artwork")
+          const _options = {
```

Comment:
@mzikherman - would it make sense to mirror the naming? `exclude` -> `exclude_id`s? Seems like less drift is better.

Reply:
Sure, although you'd need to keep `exclude` working in MP (to be backwards compatible), so are you wanting to update Gravity to use that instead of `exclude_ids`?

I'll leave it up to you, I think this is innocuous enough to leave, but I agree with your intuition that less drift is better.

Reply:
I was looking back in Gravity and I think having a more explicit API param in gravity is probably better.